Hypnotic-Herons/Sprint-2
# Md createtraining

## Description of Proposed Changes
- added a list for the training program details

## Steps to Test

Outline the steps to test

```
git fetch --all
git checkout md_createtraining
code .

```
navigate to the root
python manage.py makemigrations HRIT
python manage.py migrate
python manage.py createsuperuser ```(this is just for your local use)```
python manage.py runserver
```
1. login on the admin view http://127.0.0.1:8000/admin
1. add a training program or two
1. look at the browser view for training programs to see the list http://127.0.0.1:8000/HRIT/trainingprograms/

## Impacted Areas in Application

browser views to show training program info


## Definition of Done

1. The project must be fully documented. This includes the following:
    1. Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
    1. Every class must be documented with purpose, author, and methods.
    1. Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
1. The project must be able to run fully, and without errors, on each teammate's system.
1. Fulfills every requirement.
Every line of code has been peer reviewed.
1. For projects that require unit testing, core functionality must be identified and have at least one test for each.
